lib/pbench/test/unit/agent/task/test_move_results.py

```diff
@@ -1,15 +1,43 @@
-import os
 import datetime
+import logging
+import os
 import responses
 
+from pathlib import Path
 from pbench.cli.agent.commands.results import move_results
 from pbench.test.unit.agent.task.common import MockDatetime
 
 
+mdlog_tmpl = """[pbench]
```

Comment:
This section would be a lot easier to "grok" at a glance with a comment; and maybe even just a line break after the opening `"""` to make the bracketing more obvious.

Reply:
See comments in #1947.